packages/commercetools/theme/pages/CheckoutV2/Shipping.vue

```diff
@@ -1,13 +1,7 @@
 <template>
   <div>
-    <SfHeading
-      :level="3"
-      title="Shipping"
-      class="sf-heading--left sf-heading--no-underline title"
-    />
     <ShippingForm
       :handleShippingAddressSubmit="handleShippingAddressSubmit"
```

Comment:
submitShippingAddress 🗡️

Reply:
Okay. But I would leave props name as it is:
```vue
<ShippingForm
      :handleShippingAddressSubmit="submitShippingAddress"
    />
```
Wdyt?